src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -783,6 +784,30 @@ protected void addErrorForUpdatingItem(Piece piece, Throwable e) {
     }
   }
 
+  protected Map<String, Piece> getProcessedPiecesForPoLine(String poLineId, Map<String, List<Piece>> piecesGroupedByPoLine) {
```

Comment:
This group of 4 methods are misplaced in "Errors" section. Please check comments after dashes in this class:
![image](https://github.com/folio-org/mod-orders/assets/144257054/850d42f2-515f-4397-bb1d-c82c209a4cd9)
Actually this class is already big and hard to manage. We can easily move new methods to corresponding classes but maybe this type of refactoring increases the scope a bit.

Reply:
Will move above the errors, thanks. I guess yes, the class is pretty big and can be refactored, unsure if in scope of this.